ote/src/cljs/ote/views/transport_operator.cljs

```diff
@@ -413,34 +436,36 @@
-     [:div
-      [info/info-toggle (tr [:common-texts :instructions] true)
-       (if ytj-supported?
-         [:div
-          [:div (tr [:organization-page :help-ytj-integration-desc])]
-          [:div (tr [:organization-page :help-desc-1])]
-          [uicommon/extended-help-link (tr [:organization-page :help-about-ytj-link]) (tr [:organization-page :help-about-ytj-link-desc])]
-          [uicommon/extended-help-link (tr [:organization-page :help-ytj-contact-change-link]) (tr [:organization-page :help-ytj-contact-change-link-desc])]]
-         [:div
-          [:div (tr [:organization-page :basic-info-tooltip])]])]]
-
-     ;; When business-id has multiple companies create list of delete-operator dialogs. Otherwise add only one
-     (if (empty? (:ytj-company-names state))
-       [delete-operator e! operator (:transport-operators-with-services state)]
-       (for [o (get-in state [:transport-operator :transport-operators-to-save])]
-         ^{:key (str "operator-delete-control-" (::t-operator/name o) "-" (::t-operator/id o) )}
-         [delete-operator e! o (:transport-operators-with-services state)]))
-
-     (if show-merge-companies?
-       (operator-merge-section e! operator (:ytj-company-names state))
-       [form/form
-        (operator-form-options e! state show-details?)
-        form-groups
-        operator])]))
+       [:div
+        [:div
+         [:h1 (tr [:organization-page
+                   (if (:new? operator)
+                     :organization-new-title
+                     :organization-form-title)])]]]
+       [:div
+        [info/info-toggle (tr [:common-texts :instructions] true)
+         (if ytj-supported?
+           [:div
+            [:div (tr [:organization-page :help-ytj-integration-desc])]
+            [:div (tr [:organization-page :help-desc-1])]
+            [uicommon/extended-help-link (tr [:organization-page :help-about-ytj-link]) (tr [:organization-page :help-about-ytj-link-desc])]
+            [uicommon/extended-help-link (tr [:organization-page :help-ytj-contact-change-link]) (tr [:organization-page :help-ytj-contact-change-link-desc])]]
+           [:div
+            [:div (tr [:organization-page :basic-info-tooltip])]])]]
+
+       ;; When business-id has multiple companies create list of delete-operator dialogs. Otherwise add only one
+       (if (empty? (:ytj-company-names state))
+         [delete-operator e! operator (:transport-operators-with-services state)]
+         (for [o (get-in state [:transport-operator :transport-operators-to-save])]
+           ^{:key (str "operator-delete-control-" (::t-operator/name o) "-" (::t-operator/id o))}
+           [delete-operator e! o (:transport-operators-with-services state)]))
+
+       (if show-merge-companies?
+         (operator-merge-section e! operator (:ytj-company-names state))
+         [form/form
+          (operator-form-options e! state show-details?)
+          form-groups
+          operator])]
+      [spinner/circular-progress])))
```

Comment:
Looks like on some response completions the circular progress stays displayed and spinning, namely on `->ServerError`.  If user doesn't notice the flash toast message, it will feel like request is still ongoing.
A bit more robust design would be to have a `request-XXX-ongoing?`for front-end display logic, which in this case controls the spinner.

Alternatively in this case `country-list-loaded?` and `transport-operator-loaded?` could be toggled in the error case, but I didn't check if that would cause logical problems 
(request-completed and list-loaded are not synonymous but a bit different logically)